ob_v3p0/impl/introduction.md

```diff
@@ -0,0 +1,102 @@
+organization as Verifiable Credentials compatible with the [[vc-data-model]].
+Issuers may bundle together multiple related achievement credentials into
+transcripts and other longitudinal records for an individual learner in a CLR as
+a `ClrCredential`, which is also signed using the same technique as the
+individual credentials. Additionally, credentials can be augmented with an
+`EndorsementCredential` from a third party to lend the support of another
+individual or organization to the quality or relevance of an issuer or
+credential data.
+
+A RESTful API, with dynamic client registration, is available to transport data
+in `AchievementCredential` and `ClrCredential` format, under the control of the
+learner, between systems where they are issued, hosted on behalf of the learner,
+or verified by third parties in order to qualify the learner for job placement
+or other opportunities. Implementing systems can participate in a variety of
+roles
+
+#### Audiences
+
+This implementation guide is intended for product developers across various
+implementation roles necessary for the operation of an ecosystem where digital
+credentials efficiently recognize achievements that matter and flow to the
+contexts where these achievements each need to be understood. Products may be
+situated to perform one or more roles within the ecosystem, such as issuing
+credentials, hosting credentials on behalf of learners, and verifying
+credentials.
+
+### OB Overview
+
+An Open Badge (`AchievementCredential`) is a individual achievement recognized
+about an individual learner. An Issuer makes a claim that a learner has met the
+criteria of a particular defined `Achievement`.
+
+### CLR Overview
+
+A Compehensive Learner Record allows many Open Badge achievement credentials to
+be bundled together, with some additional associations between them defined.
+This is like another onion layer wrapping the inner set of credentials that is
+also signed. Individual component credentials are verifiable, and the wrapping
+CLR is also verifiable. CLRs can contain achievements from multiple different
+issuers to show a learner's progression with multiple organizations or
+subdivisions of a large educational institution.
+
+### Use Cases
+
+Use cases are outlined in each the Open Badges and Comprehensive Learner Record
+specifications. Use cases outline how each specification is intended to provide
+value to end users through interoperability between products.
+
+[Open Badges use cases](https://1edtech.github.io/openbadges-specification/ob_v3p0.html#use-cases)
+include:
+
+-   Assertion Issuance to Wallet
+-   Assertion Issuance Without a Wallet
+-   Recipient Presentation of Assertion
+-   License Issuance
+-   Single Skill Assertion
+-   Mapping Skills
+-   Verifying Continuing Education
+-   Self-assertion
+-   Endorsement
+-   Re-issue a <= 3.0 Badge to a 3.0 Badge
+-   Authorization to Issue Given by Creator to Issuer
+-   Revocation of an Issued Credential
```

Comment:
I'd recommend removing Revocation of an Issued Credential unless we are mentioning how revocation would work in the base spec.

Reply:
I added a note about how the importance of the use cases and how revocation might be normatively covered in a future version of the spec in commit [232fba0](https://github.com/1EdTech/openbadges-specification/pull/501/commits/232fba0fcc831be806adf3a7cd9b73196b1e6b1e). That's probably the optimal balance--I know some readers are really going to want to see something about revocation in here.